src/SignaturePad.Android/SignaturePadView.cs

```diff
@@ -46,19 +46,21 @@ public class SignaturePadView : RelativeLayout {
 				if (points == null || points.Count () == 0)
 					return new System.Drawing.PointF [0];
 
-				List<System.Drawing.PointF> pointsList = points [0].ToList ();
-				
+				IEnumerable<System.Drawing.PointF> pointsList = points[0];
+
 				for (var i = 1; i < points.Count; i++) {
-					pointsList.Add (System.Drawing.PointF.Empty);
-					pointsList = pointsList.Concat (points [i]).ToList ();
+					pointsList = pointsList.Concat (new [] { System.Drawing.PointF.Empty });
+					pointsList = pointsList.Concat (points [i]);
 				}
```

Comment:
Would it not be better to do:

    List<System.Drawing.PointF> pointsList = new List<System.Drawing.PointF>(points.Count*2);
    for (...) {
        pointsList.Add(System.Drawing.PointF.Empty)
        pointsList.Add(points [i])
    }